### ISSUE TITLE
sab 1.1.1x -- glitter on high latency connection == 'lost connection' non stop

### ISSUE THREAD

**Issue by thezoggy** [sabnzbd/sabnzbd#650]
friend accesses his sab instance from around the world (travels a lot) over 3g/lte hotspots.. a lot of time he has decent speeds but just high latency. thus glitter becomes quite annoying to use due to the 'lost connection' modal that pops up all the time.

looking through the code, it appears that the updateQueue is only given 100ms before you see the modal..  and also doesn't look like it handles if it is processing previous update was delayed, thus there is this domino effect where the previous one blocks the next one and then you get this cascading lost connection problem because of it.

https://techoctave.com/c7/posts/60-simple-long-polling-example-with-javascript-and-jquery
http://reallifejs.com/brainchunks/repeated-events-timeout-or-interval/

**Comment by Safihre** [comment]
It's given 10.000msec, see callApi 
https://github.com/Safihre/sabnzbd/blob/develop/interfaces/Glitter/templates/static/javascripts/glitter.basic.js

**Comment by Safihre** [comment]
Also it doesn't use Interval but only sets a new timeout when the previous was processed, see self.refresh here https://github.com/Safihre/sabnzbd/blob/develop/interfaces/Glitter/templates/static/javascripts/glitter.main.js

Maybe it was different in 1.0.3?

**Comment by Safihre** [comment]
Actually I see you are right, there is a cascading effect. The placement of the new timeout is in the wrong place!

**Comment by Safihre** [comment]
@thezoggy, I fixed it now so indeed only a next update will be called when the previous one was successful. However, I always tell the ```ajax``` request to have a timeout of 10.000msec, also on 1.0.3, is his connection really slower than that?
In Chrome I tried by throttling it to 3.000msec using the dev-tools, but it won't show the restarting overlay. Could it be that his browser overrides the timeout setting?
